generator_files/cookbooks/infranodes/recipes/default.rb

```diff
@@ -54,4 +56,6 @@
 node.set['push_jobs']['chef']['node_name'] = node['demo']['node-name']
 node.default['push_jobs']['allow_unencrypted'] = true
 
-include_recipe 'push-jobs'
+if node['platform'] == 'linux'
```

Comment:
Ideally this should be:
`include_recipe 'push-jobs' if node['platform'] == 'linux'`
https://github.com/bbatsov/ruby-style-guide#if-as-a-modifier

Reply:
Aaaaaandreeeeeeee!!! (fixed. :-D )